convert/cloudformation-only/main.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_iam_policy" "cloudformation_only" {
```

Comment:
Maybe it's overkill, but I feel like just creating & deleting this role in Python when `delete-stack` is called is a nicer "experience", but it's not a huge deal. It's also not a unique name, but that is only a problem for testing.

Reply:
I kind of figured that once we're creating things, it's better to have terraform with state etc. Additionally, I also made this something that can be done manually for the paranoid, so having that managed outside of the script makes sense for that.